docker-compose.yaml

```diff
@@ -210,10 +228,16 @@ services:
       - POSTGRES_DB=hive
       - POSTGRES_URL=postgres:5432
       - JUPYTERHUB_ADMIN_PASSWORD=testpassword123
+      - NETWORK_NAME=cdm-jupyterhub-network
+      - JUPYTERHUB_USER_IMAGE=cdm-jupyterhub-cdm_jupyterhub:latest
```

Comment:
You don't want to pin this?

Reply:
in Rancher, it's pined. But locally latest is the one we want to test against.

Reply:
:+1: